modules/claims_api/app/workers/claims_api/evidence_waiver_builder_job.rb

```diff
@@ -1,19 +1,36 @@
 # frozen_string_literal: true
 
 require 'sidekiq'
+require 'claims_api/ews_vbms_sidekiq'
 require 'claims_api/evidence_waiver_pdf/pdf'
 
 module ClaimsApi
   class EvidenceWaiverBuilderJob
     include Sidekiq::Worker
+    include ClaimsApi::EwsVBMSSidekiq
 
     # Generate a 5103 "form" for a given veteran.
     #
-    # @param power_of_attorney_id [String] Unique identifier of the submitted POA
-    def perform(target_veteran:, response: true)
-      waiver = ClaimsApi::EvidenceWaiver.new(target_veteran: target_veteran)
-      output_path = waiver.construct(response: response) # rubocop:disable Lint/UselessAssignment
-      # upload `output_path` here (and remove rubocop:disable above)
+    # @param evidence_waiver_id [String] Unique identifier of the submitted EWS
+    def perform(evidence_waiver_id)
+      evidence_waiver_submission = ClaimsApi::EvidenceWaiverSubmission.find(evidence_waiver_id)
+      auth_headers = evidence_waiver_submission.auth_headers
+      waiver = ClaimsApi::EvidenceWaiver.new(auth_headers: auth_headers)
+      waiver.construct
+    rescue VBMS::Unknown
+      rescue_vbms_error(evidence_waiver_submission)
+      raise
```

Comment:
we don't have these raises in PoA builder. Why did we add them here?

Reply:
but we do have them in the POA builder job: https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb#L27
right, or am I missing something?

Reply:
correct, buy we use the rescue methods, we don't additionally include a `raise` after the rescue method... 
https://github.com/department-of-veterans-affairs/vets-api/blob/17d748f40b641d4052621be60e2309daf9a1a591/modules/claims_api/app/workers/claims_api/poa_form_builder_job.rb#L28